tests/acceptance/bootstrap/CliContext.php

```diff
@@ -233,4 +234,88 @@ public function theCommandOutputShouldContain(string $shouldOrNot, string $outpu
 			Assert::assertStringNotContainsString($output, $jsonResponse["message"]);
 		}
 	}
+
+	/**
+	 * @When the administrator lists all the upload sessions
+	 * @When the administrator lists all the upload sessions with flag :flag
+	 *
+	 * @param string|null $flag
+	 *
+	 * @return void
+	 */
+	public function theAdministratorListsAllTheUploadSessions(?string $flag = null): void {
+		if ($flag) {
+			$flag = "--$flag";
+		}
+		$command = "storage-users uploads sessions --json $flag";
+		$body = [
+			"command" => $command
+		];
+		$this->featureContext->setResponse(CliHelper::runCommand($body));
+	}
+
+	/**
+	 * @When the administrator cleans all expired upload sessions
+	 *
+	 * @return void
+	 */
+	public function theAdministratorCleansAllExpiredUploadSessions(): void {
+		$command = "storage-users uploads sessions --expired --clean --json";
+		$body = [
+			"command" => $command
+		];
+		$this->featureContext->setResponse(CliHelper::runCommand($body));
+	}
+
+	/**
+	 * @Then /^the CLI response should (not|)\s?contain these entries:$/
+	 *
+	 * @param string $shouldOrNot (not|)
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 */
+	public function theCLIResponseShouldContainTheseEntries(string $shouldOrNot, TableNode $table): void {
+		$expectedFiles = $table->getColumn(0);
+		$responseBody = $this->featureContext->getJsonDecodedResponse($this->featureContext->getResponse());
+		preg_match('/(\[.*\])/', $responseBody["message"], $matches);
+		$responseArray = \json_decode($matches[1], null, 512, JSON_THROW_ON_ERROR);
```

Comment:
some comment with example here would be great to have
```suggestion
		preg_match('/(\[.*\])/', $responseBody["message"], $matches);
		$responseArray = \json_decode($matches[1], null, 512, JSON_THROW_ON_ERROR);
```